v2/fleet-local/control/zk-health@.service

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Exhibitor/Zookeeper health checker/block @ %i
+
+# forces things like the update-engine (and related services such as managed
+# CoreOS) to wait for this unit to complete
+Before=locksmithd.service marathon@%i.service mesos-master@%i.service
+After=zk-exhibitor@%i.service
+
+[Service]
+Type=simple
+User=core
+Restart=on-failure
+RemainAfterExit=true
+
+# array of objects, each representing a ZK node
+Environment="EX_QUERY=curl -sS localhost:8181/exhibitor/v1/cluster/status"
+# expected size of ensemble after it's done
+Environment="CTL_SIZE=etcdctl get /environment/CONTROL_CLUSTER_SIZE"
+
+# wait for zk-exhibitor to start - can't use %i specifier here
+ExecStartPre=/usr/bin/bash -c 'while [ "$(systemctl is-active zk-exhibitor@*)" != "active" ]; do sleep 2; done'
+# give the instance ~16s to start up & serve since the container starting doesn't mean much for curl
+ExecStartPre=/usr/bin/sleep 16
```

Comment:
This unit will restart if it does fail either way, so technically it's not needed. But I wanted to avoid having this unit restart over and over, spamming the logs post-reboot